### PR TITLE
Add new feature to project

### PR DESCRIPTION
**Pull request by Abdelaziz-Mahdi**
need a review

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+*.cab
+*.msi
+*.msix
+*.msm
+*.msp
+
+# Windows shortcuts
+*.lnk
+###a file should be ignored by git###
+test.md
```
Comment (BregornOriginal) [inline .gitignore:27]: Kindly add the `node_modules` folder to the list of ignored on the `.gitignore` file 💯 

You can do it by writing the name of the folder inside the `.gitignore` file, like this 

image

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (BregornOriginal) [inline .github/workflows/linters.yml:48]: To check the linters locally you can follow the instructions on the [Microverse documentation](https://github.com/microverseinc/linters-config/tree/master/html-css)
In this way, you can check your linters before doing a Pull Request and make your code cleaner! 🥇 

Kindly, fix these linter errors on your style.css file: 

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @Abdelaziz-Ali , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Excelent README documentation 💯
✔️ Excellent commit history 💯
✔️ Correct use of GitHub Flow 💯
✔️ Pull Request has a descriptive summary! 💯 

the-simpsons-homer-simpson

## Required Changes ♻️♻️♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
[OPTIONAL] On your README file you can improve it by adding differents points for each action! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (Abdelaziz-Mahdi) [reply .gitignore:27]: Loved that tip, Thank you.

**Review by Owusu-Desmond — COMMENTED** [review]
Hi @Abdelaziz-Ali , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

NOTE: DO NOT MERGE A PULL REQUEST UNTIL IS APPROVED :red_circle: 

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**